browser/flagr-ui/src/components/Flag.vue

```diff
@@ -236,24 +241,17 @@
                             </el-button>
                           </div>
                         </div>
-                        <el-row :gutter="10">
-                          <el-col :span="12">
-                            <el-input
-                              size="small"
-                              placeholder="Key"
-                              v-model="variant.key">
-                              <template slot="prepend">Key</template>
-                            </el-input>
-                          </el-col>
-                          <el-col :span="12">
-                            <el-input
-                              size="small"
-                              placeholder="{}"
-                              v-model="variant.attachmentStr">
-                              <template slot="prepend">Attachment </template>
-                            </el-input>
-                          </el-col>
-                        </el-row>
+                        <el-collapse class="flex-row">
+                          <el-collapse-item title="Variant attachment" class="variant-attachment-collapsable-title">
```

Comment:
just curious, if variant.attachment is not empty (null or {}), can we automatically show the attachment? Just want to highlight the visibility of non-empty attachment.

Reply:
It seems like `element-ui` does not support manually setting the `active` flag, so I think that this is not possible to use out of the box capabilities.